spec/client_spec.rb

```diff
@@ -50,3 +50,26 @@
     it { expect(result).to eq 'be26ac57ef264c23662ba373e4b68670c1a006431c763af6d33ad10ab6aa97d9' }
   end
 end
+
+describe Centrifuge do
+  context 'delegation' do
+    it 'class should delegate methods to default client' do
+      client_double = double('default_client')
```

Comment:
Test creates a double, since some methods require args, and writing separate test for each delegation would create a lot of clutter.